### DISCUSSION TITLE
1d Spring Dampers

### DISCUSSION THREAD

**Discussion by Edward-Eth** [idaholab/moose#20770] [Q&A Modules: Solid mechanics] [answered]
Hi all, simple question here. I've done a bit of digging through the source docs and have struggled to find a simple way of implementing 1D spring dampers in a model. Is there no simple inbuilt tool for this purpose? Just checking there isn't before I make one myself.

**Comment by GiudGiud** [comment, marked as answer]
@jiangwen84 @gambka any hints on this?

**Reply by GiudGiud** [reply]
@Edward-Eth in which context is this?
Is this for numerical or physical relaxation? Seems like the latter.

Are they going to be attached to DOFs ? How many?

**Reply by Edward-Eth** [reply]
This is as a simplification of a physical component in a structure. I suppose they'd be attached to displacements and velocities between nodes. I'm suspecting that this could actually be achieved by the tactical application of DynamicTensorMechanics to 1D mesh elements.

**Reply by GiudGiud** [reply]
not sure.
This seems like an ODE that would live on a single node to me. PLease have a look at the NodalKernel system and let me know if it seems like the way to do this
https://mooseframework.inl.gov/syntax/NodalKernels/index.html
